### PR TITLE
Quality checks

### PR DESCRIPTION
**Pull request by Sloy**
Added new formatting standard from Schibsted Spain, Checkstyle, PMD and Findbugs.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ android:
 
 install: /bin/true
 
-script: ./gradlew build
+script: ./gradlew check build
```
Comment (CristianGM) [inline .travis.yml]: You don't need to call check, build already does it.

File: build.gradle [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+  }
+  dependencies {
+    classpath 'com.android.tools.build:gradle:2.1.2'
+    classpath 'com.novoda:bintray-release:0.3.4'
 
-        // NOTE: Do not place your application dependencies here; they belong
-        // in the individual module build.gradle files
-    }
+    // NOTE: Do not place your application dependencies here; they belong
+    // in the individual module build.gradle files
```
Comment (CristianGM) [inline build.gradle]: would yo mind to clean this useless comment?

**Comment by CristianGM** [comment]
Your build failed, maybe you should take a look to this: http://stackoverflow.com/questions/35634437/travis-ci-build-failing-cause-permgen-space

File: .travis.yml (hunk 1 above)
Comment (Sloy) [inline .travis.yml]: U sure? I know it does some checks, but I wasn't sure it ran the whole check task. I'll remove it then.

File: build.gradle (hunk 2 above)
Comment (Sloy) [inline build.gradle]: I will gladly do that, sir

**Comment by Sloy** [comment]
Thanks @CristianGM, check it out now

**Comment by CristianGM** [comment]
😍 👍

**Pull request merged**